### ISSUE TITLE
device option

### ISSUE THREAD

**Issue by rmarabini** [HeliXonProtein/OmegaFold#42]
Hi,

   Could you help me to understand how the option "device" work?  
Should OmegaFold run on GPU is I write "--device cuda"? 
How can I choose a particular GPU? 

Right now Omegafold is running in my workstation using 32 cores and ignoring the GPUs.

    thanks in advance

**Comment by rpkarandev** [comment]
Yes, I could confirm that. OmegaFold used GPU with explicit use of "--device cuda". 
"cuda" in lowercase
